### ISSUE TITLE
Suggestion to give more feedback to the user

### ISSUE THREAD

**Issue by GitBoudewijn** [dsohler/oneclick-downloader#10]
Hey,

I already mentioned some of this in another thread but I thought I would make a new one so it's more organized. I think it would be helpful if the extension gave more information to the user about the progress of the steps. Here is what I would like to see:

First step:
Show how many tabs it has found and how many tabs you currently have selected. If no tabs are found, show a message that says something like "No suitable tabs found. Open the tabs you want to download from and click rescan.". Disable the scrape button if nothing is selected.

Second step:
Show how many links it has found and how many links you currently have selected. If no links are found, show a message that says something like "No supported links found. If you want to add support for a certain link, go to github.com/dsohler/oneclick-downloader/issues.". Disable the download button if nothing is selected.

Third step:
Show how many files it has downloaded and how many there are still remaining. If possible also show a timer that shows how long the downloads are running and maybe also a Windows style time remaining prediction (might be too complicated:P). It would also be cool if it included this information in the title attribute, so you can minimize the window and see how many files are left for example.

(I hope you don't think I'm too obsessed with usability.:P)

**Comment by dsohler** [comment]
Yes, we totally need a status bar. Since status bars in browser windows are no longer a thing we need to implement one manually. I was already thinking of it. The best thing would be combining the buttons and some kind of status indicator in a fixed area at the bottom of the window. The fixed area isn't a problem because the buttons can easily be moved into their own section

Since XHR works in an asynchronous way and currently all file URLs are simply sent to the Chrome download manager it's not that easy to properly monitor the download process. There are some functions but it seems too overcomplicated to re-implement chrome://downloads within the extension :smile:

What would be possible is to show the amount of running, remaining, and failed downloads in the planned status bar. The downloading process could be added to the title, too. So when minimizing it you can stil hover the taskbar button to see the process.

Before implementing this I want to force the downloading process into some kind of "pseudo-synchronous" processing to control how many downloads at the same time are running.
